### PR TITLE
Task3 components1

### PR DESCRIPTION
**Pull request by SteveRusin**

### CODE REVIEW

File: src/home-page/Content/Controls/CategoryControls.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { ButtonBase } from '@shared';
+
+export const CategoryControls = () => {
+  return (
+    <>
```
Comment (ygorbunkov) [inline src/home-page/Content/Controls/CategoryControls.tsx:28]: This doesn't seem to be related to your `MovieGenre` enum. To stay consistent, I would rather keep these connected:
```jsx
return Object
   .values(MovieGenre)
   .map(genre => (
      <ButtonBase key={genre} className='category-button'>{genre}</ButtonBase>
   ))
```

Comment (SteveRusin) [reply src/home-page/Content/Controls/CategoryControls.tsx:28]: Yeah, probably. I think it will be implemented as a part of one of next HW
